### DISCUSSION TITLE
How does this example work with a single line JSON?

### DISCUSSION THREAD

**Discussion by hotzenwalder** [Weatherlights/Winget-AutoUpdate-Intune#25] [Q&A]
On this link you can find an example of a mod for the installation of Zoom
https://github.com/Romanitho/Winget-AutoUpdate/discussions/521

I understand that with this fork you can use a modifcation with a JSON oneliner, but I struggle to grasp the syntax.
How would the example above for Zoom work with a single line JSON?

Also... when you want to close multple processes prior to updating or installing an application, how would that work?

**Comment by Weatherlights** [comment]
This mod is currently not supported in this release. However I already implemented the code and will be in there in future releases (that you can add this registry key). However you can also set this registry key by using an ADMX backed policy. There is another project of mine: PolicyApplicator which allows you to capture configurations of existing apps and apply them independently from the installation.
